documentation/SOFTWARE_REQUIREMENTS_SPEC.md

```diff
@@ -47,13 +47,15 @@ Our web-based Sokoban game will have greater user reachability compared to a mor
 
 ### Phase 2: New Game Features (Optional)
 * The game should have multi-player option.
-* Each level should have additional element added to the world. For example, conveyor belts, teleporters, boxes that decay when pushed, ice floors that boxes slide on, multiple players that move in tandem, etc.
-* AI generated optimized solution. With the given world, the game should have a hint feature showing the optimal moves to solve the world.
+* Each level should have additional element added to the board. For example, conveyor belts, teleporters, boxes that decay when pushed, ice floors that boxes slide on, multiple players that move in tandem, etc.
+* AI generated optimized solution. With the given board, the game should have a hint feature showing the optimal moves to solve the world.
```

Comment:
See discussion in another thread. Even marking this optional seems out of scope, unless we plan to use an external library--I suggest we pick lower-hanging fruit here, or at least acknowledge the difficulty of the problem.